hsrc_lib/Database/LMDB/Raw.hs

```diff
@@ -1,4 +1,9 @@
-{-# LANGUAGE ForeignFunctionInterface, DeriveDataTypeable #-}
+{-# LANGUAGE DeriveDataTypeable    #-}
+{-# LANGUAGE DerivingVia           #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE LambdaCase            #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{- HLINT ignore "Use camelCase" -}
```

Comment:
Probably define in a `.hlint.yaml` file? it looks reasonable if dealing with FFI

Reply:
It's only for this file, so I'll probably leave it as is